pgxn/neon/control_plane_connector.c

```diff
@@ -508,6 +511,8 @@ NeonXactCallback(XactEvent event, void *arg)
 static bool
 RoleIsNeonSuperuser(const char *role_name)
 {
+	assert(role_name);
```

Comment:
This is the C99 assert function, that's not used anywhere currently. Should be `Assert`, for the Postgres macro.
```suggestion
	Assert(role_name);
```

Reply:
I was thinking there was one, but forgot what it was called :laughing: